sentences/fr/media_player_HassMediaNext.yaml

```diff
@@ -0,0 +1,8 @@
+language: fr
+intents:
+  HassMediaNext:
+    data:
+      - sentences:
```

Comment:
I think we could have more sentences such with verbs
- Passe / Passer au morceau suivant sur la TV
- Met / mettre le morceau suivant sur les enceintes